utils/src/config.rs

```diff
@@ -146,6 +173,21 @@ pub const fn default_heartbeat_interval() -> Duration {
     Duration::from_millis(300)
 }
 
+/// default batch timeout
+#[must_use]
+#[inline]
+pub const fn default_batch_timeout() -> Duration {
+    Duration::from_millis(1)
```

Comment:
Is 1 millis too extreme? It will result in a busy leader. If our default heartbeat timeout is 300ms, perhaps 50ms~100ms would be more suitable?

Reply:
I think 50ms ~ 100ms may be too large. The original default value is 15ms. But in the benchmark practice, 1ms is better than 15ms in the conflict test case, saving about 20ms latency per wait synced request. And in the non-conflict test case, the performance is also quite good.

Reply:
If we want the wait sync to be as fast as possible, then I guess we don't need the batch from the beginning? We can just send out log entry as soon as possible. I guess batching is somehow a trade off between sync speed and system stability. With batch, we can have fewer append_entries sent, which can possibly result in a higher throughput when there is little conflict between cmds.

So I guess the question here is should we still take sync speed as a criteria of our benchmark? With batching, we indeed have a lower sync speed then etcd, but we can have a higher throughput when there's no conflict. If this reasoning stands, than we can have a longer `batch_timeout`.

Reply:
Actually, I tried to send an `ae` immediately when the curp realize that the proposal conflicted with the proposal it has received before. But the benchmark result is not good.